game/src/main/java/net/minecraftnt/server/entities/Entity.java

```diff
@@ -29,8 +36,32 @@ public PhysicsBody addPhysicsBody() {
     public Entity(Vector3 pos){
         this.transform = new Transform(pos);
     }
+    
+    public void setPosition(Vector3 pos) {
+    	this.transform.location = pos;
+    	this.createAABB(boundsX, boundsX);
```

Comment:
boundsX, boundsX should be boundsX, boundsY right?

I just want to ensure it all works properly before merging lol